man/rgbasm.5

```diff
@@ -487,10 +487,12 @@ is in.
 .Ar symbol
 must have been defined already.
 .It Fn SIZEOF arg Ta Returns the size of the section named
```

Comment:
How about: "If *arg* is a string expression, returns the size of the section named *arg*; if *arg* names a section type, returns the size of that section type."

I think that reads better.

Reply:
Sure. I edited it a bit to match to wording of `BANK` above.